Formula/cmusfm.rb

```diff
@@ -0,0 +1,76 @@
+class Cmusfm < Formula
+  desc "Last.fm standalone scrobbler for the cmus music player"
+  homepage "https://github.com/Arkq/cmusfm"
+  url "https://github.com/Arkq/cmusfm/archive/v0.3.3.tar.gz"
+  sha256 "9d9fa7df01c3dd7eecd72656e61494acc3b0111c07ddb18be0ad233110833b63"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+
+  libfaketime = Formula["libfaketime"]
```

Comment:
It seems like this could just be `depends_on "libfaketime" => test if MacOS.version >= :sierra`.

Reply:
It could, but who'll remember to update this constraint when libfaketime's requirements change?

Reply:
The test will fail if libfaketime's requirements tighten, but who'll remember to relax this constraint if upstream patch in compatibility for El Capitan? The reason we have this apparatus in the first place is to avoid unnecessary sleeping during builds, per your previous review.

Reply:
@puxlit El Capitan is no longer officially supported by homebrew, so is it possible to make a simple ( `=> :test`) dependency on `libfaketime`? It's only needed by the tests, correct?